config/routes.rb

```diff
@@ -1,2 +1,3 @@
 Rails.application.routes.draw do
+  resources :quotes, only: [:show]
```

Comment:
I see that you chose to NOT use any namespaces on this API application. Can you give me a reason for that? Not a blocker, I just want to know.

Reply:
We decided to go for api but we forgot to update it, our bad... The front end is already using the api route